TA Signals/StocksDataFetcher.swift

```diff
@@ -16,12 +16,16 @@ class StocksDataFetcher {
     // The variable stocks contains an array of Stock objects
     @Published var stocks = [Stock]()
     
+    var _feature_flag_json_includes_date: Bool = false
```

Comment:
`_feature_flag_json_includes_date`

FYI, it isn't usually very swifty to add an _ at the beginning of a variable name.  You will see this in some older projects that include objective C, but not in the swift code.  

The reason you don't want to use underscore requires a bit of objective c understanding.  When you declare a property in objective C, you get a few things automatically, a getter method, a setter method, and an instance variable property.

Generally in objective c when you are accessing properties in a class you are accessing it via the memory address not the property it's self.  That is what the asterisk in the objective c declaration is, you are creating a pointer at a certain location and then using that pointer to access the property.
 
When the instance variable is created for a store property it is prefixed with an underscore like:

`_identifier`

It is generally not advised to access the property directly in most cases.  

This is NOT the case in swift, but because of the heritage of variable names in the objc I would avoid using this as a naming convention when working in swift.

Reply:
Thanks for the input! I believe my usage of underscores come from projects in C#/Unity where naming private variables with starting `_` was a convention, then I brought it over Swift :sweat_smile: . Changed!